src/components/organisms/AdminVenueView/components/Spaces/Spaces.scss

```diff
@@ -94,69 +95,13 @@ $icon-size: 20px;
     border-top: 1px solid opaque-white(0.2);
   }
 
-  &__home-button {
-    cursor: pointer;
-    display: flex;
-    width: $button-size;
-    height: $button-size;
-    border-radius: 100%;
-    justify-content: center;
-    align-items: center;
-    transition: all 400ms $transition-function;
-    background-color: lighten($color: $saturated-black, $amount: 10);
-
-    &:hover {
-      background-color: lighten($color: $saturated-black, $amount: 20);
-    }
-  }
-
   &__nav-buttons {
     display: flex;
-    flex-direction: row;
-  }
-
-  &__back-button {
-    cursor: pointer;
-    display: flex;
-    height: $button-size;
-    font-weight: $font-weight--400;
-    font-size: $font-size--xs;
-    border-radius: $border-radius--xl;
-    justify-content: center;
-    align-items: center;
-    padding: $spacing--md $spacing--lg;
-    margin: 0 5px;
-    transition: all 400ms $transition-function;
-    background-color: lighten($color: $saturated-black, $amount: 10);
-
-    &:hover {
-      background-color: lighten($color: $saturated-black, $amount: 20);
-    }
-  }
-
-  &__next-button {
-    cursor: pointer;
-    display: flex;
-    height: $button-size;
-    font-weight: $font-weight--400;
-    font-size: $font-size--xs;
-    border-radius: $border-radius--xl;
-    justify-content: center;
-    align-items: center;
-    padding: $spacing--md $spacing--lg;
-    margin: 0 5px;
-    transition: all 400ms $transition-function;
-    background-color: $primary;
-
-    &:hover {
-      background-color: lighten($color: $primary, $amount: 10);
-    }
+    width: 40%;
```

Comment:
NOTE: This value is determined as a rule of thumb and it's OOS to this PR to try and generalize all different Admin panels' sizes at this time, although some stubs already exist in `constsnts.scss` as in 

```scss
$map-sidebar-width: 20%;
$chat-sidebar-width: 360px;
$navbar-height: 66px;
$footer-height: 30px;
$chat-input-height: 114px;
$announcement-banner-height: 50px;
$admin-spaces-sidebar-width: 20%;
```